### PR TITLE
Jakarta JSP 3.1 Plan Review

### PR DESCRIPTION
**Pull request by markt-asf**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [x] Declares optional features if there are any (Mark with **N/A** and check if none)
- [x] Includes minimum Java SE version
- [x] Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [x] Link to updated release record

### CODE REVIEW

**Review by keilw — CHANGES_REQUESTED** [review]
@markt-asf The readme file has both Release Review and Plan Review result table, shouldn't they be commented out for now, and just the plan one, at least that's how Authorization 2.1 or Authentication 3.0 (see https://github.com/jakartaee/specifications/pull/356) looked like.

**Comment by keilw** [comment]
@kwsutter Or is there a certain freedom to leave those and e.g. fix it with the ballot result? If it's OK like that I would start the ballot tomorrow.

**Review by keilw — APPROVED** [review]

**Comment by keilw** [comment]
I am starting the ballot but I noticed something strange under https://deploy-preview-366--jakartaee-specifications.netlify.app, the page looks weird, @markt-asf or @kwsutter, @ivargrimstad could you have a look at that please, I don't think it directly affects this PR so ballot seems OK but it would be nice to correct the formatting of the preview content.

File: pages/3.1/_index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+title: "Jakarta Server Pages 3.1 (under development)"
+date: 2021-10-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Server Pages defines a template engine for web applications that supports mixing of textual content
+(including HTML and XML) with custom tags, expression language, and embedded Java code, that gets compiled
+into a Jakarta Servlet.
+
+* [Jakarta Server Pages 3.1 Release Record](https://projects.eclipse.org/projects/ee4j.jsp/releases/3.1.0)
+  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
```
Comment (kwsutter) [inline pages/3.1/_index.md:11]: This should be updated to Jakarta EE 10 with a blank url reference (or extrapolate to the expected location for the Jakarta EE 10 release plan).

**Review by kwsutter — APPROVED** [review]
Thanks, @markt-asf!  Just a nit update -- should clean this up before we merge.

**Comment by kwsutter** [comment]
> I am starting the ballot but I noticed something strange under https://deploy-preview-366--jakartaee-specifications.netlify.app, the page looks weird, @markt-asf could you have a look at that please, I don't think it directly affects this PR so ballot seems OK but it would be nice to correct the formatting of the preview content.
> @kwsutter, @ivargrimstad is it possible that preview was affected by other bad or malformed pages around a month ago?

I agree that the Preview is messed up.  Something must have happened during this pre-deploy step.  I was still able to traverse the pages and find the material.  Although it's not formatted correctly, the data looks good.  Just a hiccup with the deployment processing...

File: pages/3.1/_index.md (hunk 1 above)
Comment (markt-asf) [reply pages/3.1/_index.md]: Done.

**Comment by keilw** [comment]
@kwsutter @ivargrimstad Merged, is it better to sort out the preview now, or could I still push/merge the ballot result first?

**Pull request merged**